client/app/bundles/course/assessment/submission/components/ScribingView/ScribingToolbar.jsx

```diff
@@ -727,93 +722,91 @@ class ScribingToolbar extends Component {
             placement="top"
             title={<FormattedMessage {...translations.select} />}
           >
-            <Icon
-              className="fa fa-mouse-pointer"
+            <IconButton
               color={
                 this.props.scribing.selectedTool === scribingTools.SELECT
                   ? 'primary'
                   : undefined
               }
               onClick={this.onClickSelectionMode}
               style={styles.tool}
-            />
+            >
+              <NorthWest />
```

Comment:
Find better icon

Reply:
I have tried to find the one closely resembling the mouse cursor, but seems like MUI icon does not have that. I will try finding it again later

Reply:
Done